README.md

```diff
@@ -1 +1,59 @@
-# intelligent_placer
\ No newline at end of file
+# Intelligent Placer
+## Project description
+Intelligent placer is a software for placing objects in polygon. 
+Programm gets input picture like this:
+![test 10](/test/img/10.jpg "test 10")
+And if exist disposition like this:
+![test 10 proof](/item_set/a.jpg "test 10 proof")
+where all objects placed in polygon, programm answer true, otherwise - false.
+
+Now this is just educational task, so programm can receive only photos of objects, which were in training dataset.
+
+## Formal statement of the problem
+### General
+- programm gets input with path to photo of objects and polygon(all objects and polygon on one photo)
+- programm must answer and stop working in no more than *5 minutes*
+- answer must be *"true"* or *"false"*
+- answer output to standart output stream (in console)
+
+### Task terms
+- polygon must be closed
+- polygon must be convex
+- objects are unique (no instances of same object)
+- programm answer *"false"* if input incorrect
+- programm may output warning, that input was incorrect, but its optional
+- if object is not solid, calculations anyway treat them like it is solid in position that was on photo. For examle:
+    - plush toy
+    - rope
+    - chain
+    - scissors
+- all objects treat as continuous, with no holes
+    - if object has holes, they treated as interior of objects, not as available place
+    - if in input photo was object *A* into hole of object *B*, *A* will not be consider as independent object
+
+### Content of photo
+- objects and polygon must be placed on white (or light) surface
+- objects are from training dataset
+- objects must not collide with each other
+    - at least 4 pixels between objects must contain only pixels of surface, no objects pixels
+- objects must not collide with polygon
+    - at least 4 pixels between objects and polygon must contain only pixels of surface, no objects or polygon pixels
+- objects and polygon must be entirely contained in a photo
+    - at least 4 pixels on boards of photo must contain only pixels of surface, no objects or polygon pixels
+- objects must be placed outside of the polygon
+- polygon must be black (or dark)
+
+### Photometric
+- photo must be in *\*.jpg* format
+- camera direction can be rotated no more then *15&deg;* from surface macronormal (vertical in common case)
+- illumination and color of objects should be such that they contrast with surface
+    - object border at least 10 units darker than surface (if lightness is sum of components in R8G8B8 colorspace)
+- other photometric parameters are not specified, but photo still should meet other types of requirements
+
+
+## Chosen set of items
```

Comment:
Dataset is good. I have a question about a plush toy. Are you going to position as in the photo or permanent position? I ask because in some of the photos the positions of some parts of the toy (tail) are different.

Reply:
position does not have to be permanent from one test to another, but while place program threat object as solid, so position in of tail relative to the body in hypothetical placements is same as input photo